public/js/services/routingMiddleware.js

```diff
@@ -40,11 +40,10 @@ export const updateStateFromUrlChangeMiddleware = ({ dispatch, getState }) => (n
             ...filterObj
         };
 
-        // In practice this stops react-router running the filters unless it's a
-        // page load as these should never not be equal during this action
-        // TODO: get the router info through other means on page load
```

Comment:
Does this mean it's done? 😃

Reply:
Not quite "done" but made it more explicit with the `loaded` flag on the state ... 50% done ;)

Reply:
Done-ish. Definitely a step in the right direction 👍